includes/chartability.JSON

```diff
@@ -649,4 +649,4 @@
       "POUR Coding": "perceivable, robust",
       "Additional Category Coding": "flexible, presentation, understandable, alternative, user-controlled, respectful, empowering"
     }
-   ]
\ No newline at end of file
+   ]
```

Comment:
I think GitHub's in-browser editor added this new line. Should we leave it? If not, I can remove